tools_docker/test_docker/setup_test.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+pip install pytest testinfra python-dotenv
+cp tracim.sample.env tracim.env
+cp test_config.sample.env test_config.env
+docker run --name dns_docker_solver --hostname dns.mageddo --restart=unless-stopped -p 5380:5380 \
+-v /var/run/docker.sock:/var/run/docker.sock \
+-v /etc/resolv.conf:/etc/resolv.conf \
+defreitas/dns-proxy-server
```

Comment:
Would it make sense to tell people to just run this script instead of running these commands in the documentation?

Reply:
I don't think so, devs may need to rerun the `dns_docker_solver` container after and so need to know this container.